src/Models/IoTWorkbenchProjectBase.ts

```diff
@@ -270,4 +274,21 @@ export abstract class IoTWorkbenchProjectBase {
   }
 
   abstract async handleLoadFailure(): Promise<boolean>;
+
+  static async generateIotWorkbenchProjectFile(

```

Comment:
no need to be static

Reply:
In order to be called in `RaspberryPiDevice.ts`:  
`await IoTWorkbenchProjectBase.generateIotWorkbenchProjectFile(...);`